vispy/scene/widgets/widget.py

```diff
@@ -134,7 +140,7 @@ def inner_rect(self):
         this rectangle.
         """
         m = self.margin + self._border_width + self.padding
-        if not self.border_color.is_blank:
+        if not self._border.border_color.is_blank:
```

Comment:
This is an error (mine) -- the border width is already added in to `m`; no reason to add 1 again.